Models/testmodel1/config.hpp

```diff
@@ -0,0 +1,8 @@
+#include "global_config.hpp"
+
+extern Motor_t motors[];
+extern const uint8_t NUM_MOTORS;
+
+//TODO: Combine flightmode classes with AM control algorithms
```

Comment:
Looks like there is some stuff that was defined in config.cpp that isn't declared here.